client/components/timezone/index.jsx

```diff
@@ -60,16 +61,24 @@ class Timezone extends Component {
 	}
 
 	render() {
-		const { selectedZone } = this.props;
+		const { className, selectedZone } = this.props;
 		return (
-			<select onChange={ this.onSelect } value={ selectedZone || '' }>
+			// Changing this to onBlur will not work, so we ignore this
+			// warning.
+			/* eslint-disable jsx-a11y/no-onchange */
+			<select
+				className={ classnames( 'timezone__component', className ) }
```

Comment:
I've never seen using `__component` in a selector. Our [CSS coding guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md) suggest that the part after the double underscores define the element, so I'd rather use `__main` instead. In some cases it makes sense to use `__wrapper`, but this is not really a wrapper, so maybe `__main` makes more sense. 

What do you think?

Reply:
I'm still getting to know the selector naming rules–they're a bit different from what I'm used to 😄 

I probably just messed up; I'll have another look at the rules and try to come up with a better name.